.github/workflows/bumpup.yml

```diff
@@ -0,0 +1,51 @@
+name: Bump up gem version
+
+on:
+  workflow_dispatch:
+    inputs:
+      gem-version:
+        description: Gem version
+        required: true
+        type: string
+      email:
+        description: Your email
+        required: true
+        type: string
+
+jobs:
+  bump_up_gem_version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
Can I use the latest version of [actions/checkout](https://github.com/marketplace/actions/checkout)?

Reply:
Fix: [9f9fb4a](https://github.com/kufu/jpostcode-rb/pull/567/commits/9f9fb4ae1b83888b0f3e7433dcfe74455d876550)